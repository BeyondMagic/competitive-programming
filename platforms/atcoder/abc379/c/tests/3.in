2000000001 1
1
2000000001
