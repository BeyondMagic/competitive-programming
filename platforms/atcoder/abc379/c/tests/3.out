2000000001000000000
